src/hash_map.c

```diff
@@ -230,6 +230,19 @@ static rcutils_ret_t hash_map_check_and_grow_map(rcutils_hash_map_t * hash_map)
   return ret;
 }
 
+// Modified from http://graphics.stanford.edu/~seander/bithacks.html#RoundUpPowerOf2
+static size_t next_power_of_two(size_t v)
+{
+  size_t shf = 0;
+  v--;
+  for (size_t i = 0; shf < sizeof(size_t) * 4; ++i) {
+    shf = (1 << i);
```

Comment:
```suggestion
    shf = (((size_t) 1) << i);
```

Windows was giving: "'<<': result of 32-bit shift implicitly converted to 64 bits (was 64-bit shift intended?)"